includes/Admin/Settings.php

```
@@ -123,8 +126,21 @@
 
         $settings = [];
 
+        // Ensure new settings are populated from legacy if currently blank
+        $this->ensure_new_settings_populated_from_legacy();
+
+        // Build legacy-style values from the new settings storage.
+        $legacy_from_new = $this->get_legacy_values_from_new();
+
         foreach ( $this->get_settings_sections() as $key => $section ) {
```

PHPMD (2.15.0)

Avoid unused local variables such as '$key'. (undefined)

(UnusedLocalVariable)